spec/support/vcr_cassettes/evss/documents/get_claim_documents.yml

```diff
@@ -0,0 +1,1526 @@
+---
```

Comment:
We should trim up this cassette so it's not so huge.
I think if you search for the substring, `"fileName":""},`, you'll be able to identify all of the different elements in the `documents` attribute.  It looks like there's around 1400+ elements in that array.  We should prob trim it down to like 1 or 2.